qemu/tests/cfg/live_backup.cfg

```diff
@@ -38,3 +38,12 @@
                     granularity = 32768
                 - 131072:
                     granularity = 131072
+        - data_plane:
+            only Host_RHEL.m7
+            only virtio_blk virtio_scsi
+            iothreads = iothread0
+            virtio_blk:
+                blk_extra_params_image1 = "iothread=iothread0"
+            virtio_scsi:
+                no Host_RHEL.m7.u0, Host_RHEL.m7.u1, Host_RHEL.m7.u2
+                bus_extra_params_image1 = "iothread=iothread0"
```

Comment:
@pezhang You are right, but I just don't like use ${} in cfg file,  so keep it like this :) Thanks.